psi4/src/psi4/libpsi4util/PsiOutStream.cc

```diff
@@ -51,6 +51,8 @@ PsiOutStream::PsiOutStream(std::string fname, std::ios_base::openmode mode) {
         is_cout_ = false;
     }
 
+    // This is sort of big, but vsnprintf does not appear to work correctly on all OS's.
+    // Im looking at YOU CentOS
     buffer_.resize(512000);
```

Comment:
Probably doesn't need to be this large, but at 0.5 MB it is probably not hurting anything incase someone does a massive print statement. The `vsnprintf` could be related to [this](https://stackoverflow.com/questions/29369974/vsnprintf-with-s-segfaults-with-gcc).